gdx/src/com/badlogic/gdx/graphics/glutils/ShaderProgram.java

```diff
@@ -210,13 +210,13 @@ private int loadShader (int type, String source) {
 
 		int compiled = intbuf.get(0);
 		if (compiled == 0) {
-// gl.glGetShaderiv(shader, GL20.GL_INFO_LOG_LENGTH, intbuf);
-// int infoLogLength = intbuf.get(0);
-// if (infoLogLength > 1) {
+			// gl.glGetShaderiv(shader, GL20.GL_INFO_LOG_LENGTH, intbuf);
```

Comment:
I personally see little value in *not* completely removing these comments, but this might be better for another PR.